packages/radix-ui-themes/src/components/checkbox-card-group.css

```diff
@@ -0,0 +1,163 @@
+@import './base-checkbox.css';
+
+.rt-CheckboxCardGroupRoot {
+  line-height: var(--line-height);
+  letter-spacing: var(--letter-spacing);
+  user-select: none;
+  cursor: default;
+}
+
+.rt-CheckboxCardGroupItem {
+  min-width: 0;
+
+  display: flex;
+  align-items: center;
+  gap: var(--checkbox-card-group-item-gap);
+  padding: var(--checkbox-card-group-item-padding);
+  border-radius: var(--checkbox-card-group-item-radius);
+
+  /* cursors */
+  cursor: var(--cursor-button);
+  &:where(:has(:disabled)) {
+    cursor: var(--cursor-disabled);
+  }
```

Comment:
This isn't working for some reason, because of how the CSS gets reordered.
It's making me wonder if we have the same issue in the similar places where you've done that in your last PR?